netlify/functions/spotify-control.js

```
@@ -0,0 +1,485 @@
+
+  try {
+    await spotifyRequest(username, '/me/player/next', { method: 'POST' });
+
+    // Get next track info and update session
+    if (sessionId) {
+      // Wait a moment for Spotify to update
+      await new Promise(resolve => setTimeout(resolve, 1000));
+      
+      const current = await spotifyRequest(username, '/me/player/currently-playing');
+      
+      if (current && current.item) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)